src/lwc/utilAlertBanner/utilAlertBanner.js

```diff
@@ -15,6 +15,8 @@ export default class AlertBanner extends LightningElement {
     @api icon;
     @api size;
     @api message;
+    @api isDismissable = false;
+    @api isVisible = false;
```

Comment:
@dfuller114 , when there is an error for the credit card info on the RD entry form, the error on the alert banner is not displayed in the Elevate widget (above the card holder name field). It is due to `isVisible` being false by default. Can you please check into when you have time? 
<img width="828" alt="Screen Shot 2020-12-17 at 11 07 15 AM" src="https://user-images.githubusercontent.com/28743286/102531959-18a99a00-4058-11eb-8f1a-b32245c4af7a.png">

Reply:
Thanks for the change @dfuller114 , the alert banner is displayed on the RD entry form.
<img width="781" alt="Screen Shot 2020-12-17 at 11 27 59 AM" src="https://user-images.githubusercontent.com/28743286/102534292-61168700-405b-11eb-85ec-37131de7d283.png">